.github/workflows/c-cpp.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: install packages
-      run: brew install autoconf automake
+      run: brew install autoconf automake llvm
```

Comment:
remove this llvm thing - we do not need to add this bulky package for the sake of a simple example